backoffice/templates/backoffice/vehicles/details.html

```diff
@@ -0,0 +1,96 @@
+      {{ most_recent.soc.value }}%
+    </div>
+  </div>
+</div>
+<h3 class="text-xl font-semibold text-gray-900 m-8">Previous, Current and Next Bookings for {{ vehicle.registration }}</h3>
+{% include "backoffice/bookings/bookings_list.html" with bookings=bookings %}
+
+<h3 class="text-xl font-semibold text-gray-900 m-8">Most Recent Telemetry Data</h3>
+<table class="my-8">
+    <thead>
+        <tr class="border-b border-t border-gray-200 bg-gray-50 px-6 py-3 text-left text-sm font-semibold text-gray-900">
+            <th class="border-b border-gray-200 px-6 py-3 text-left text-sm font-semibold text-gray-900 w-6">Telemetry</th>
+            <th class="border-b border-gray-200 px-6 py-3 text-left text-sm font-semibold text-gray-900 w-5">Value</th>
+            <th class="border-b border-gray-200 px-6 py-3 text-left text-sm font-semibold text-gray-900 w-8">Age</th>
+        </tr>
+    </thead>
+    <tbody class="divide-y divide-gray-100 bg-white">
+        <tr>
+            <td class="whitespace-nowrap px-6 py-3 text-left text-sm text-gray-500">Doors Locked</td>
+            <td class="whitespace-nowrap px-6 py-3 text-left text-sm text-gray-500">{{ most_recent.doors_locked.value }}</td>
+            <td class="whitespace-nowrap px-6 py-3 text-left text-sm text-gray-500">{{most_recent.doors_locked.age}}</td>
+        </tr>
+        <tr>
+            <td class="whitespace-nowrap px-6 py-3 text-left text-sm text-gray-500">Battery</td>
+            <td class="whitespace-nowrap px-6 py-3 text-left text-sm text-gray-500">{{ most_recent.battery.value }}v</td>
+            <td class="whitespace-nowrap px-6 py-3 text-left text-sm text-gray-500">{{most_recent.battery.age}}</td>
+        </tr>
+        <tr>
+            <td class="whitespace-nowrap px-6 py-3 text-left text-sm text-gray-500">Charge</td>
+            <td class="whitespace-nowrap px-6 py-3 text-left text-sm text-gray-500">{{ most_recent.soc.value }}%</td>
+            <td class="whitespace-nowrap px-6 py-3 text-left text-sm text-gray-500">{{most_recent.soc.age}}</td>
+        </tr>
+        <tr>
+            <td class="whitespace-nowrap px-6 py-3 text-left text-sm text-gray-500">Free Heap</td>
+            <td class="whitespace-nowrap px-6 py-3 text-left text-sm text-gray-500">{{ most_recent.free_heap.value }}</td>
+            <td class="whitespace-nowrap px-6 py-3 text-left text-sm text-gray-500">{{most_recent.free_heap.age}}</td>
+        </tr>
+        <tr>
+            <td class="whitespace-nowrap px-6 py-3 text-left text-sm text-gray-500">Box Uptime</td>
+            <td class="whitespace-nowrap px-6 py-3 text-left text-sm text-gray-500">{{ most_recent.uptime.value }}</td>
+            <td class="whitespace-nowrap px-6 py-3 text-left text-sm text-gray-500">{{most_recent.uptime.age}}</td>
+        </tr>
+        <tr>
+            <td class="whitespace-nowrap px-6 py-3 text-left text-sm text-gray-500">Miles Driven</td>
```

Comment:
Let's call this `Odometer` instead just to be really clear where it's coming from.